addons/mail/models/mail_thread.py

```diff
@@ -4699,6 +4699,12 @@ def _thread_to_store(self, store: Store, fields, *, request_list=None):
                 res["suggestedRecipients"] = thread._message_get_suggested_recipients(
                     reply_discussion=True, no_create=True,
                 )
+            if request_list and "contact_fields" in request_list:
+                # sudo : Returning a field name, not data
+                res["primary_email_field"] = (
+                    self.env[self._name].sudo()._mail_get_primary_email_field()
+                )
+                res["partner_fields"] = self.env[self._name].sudo()._mail_get_partner_fields()
```

Comment:
:thinking: don't think you need sudo to get field names, it's not going to try to fetch anything normally

Reply:
I remember running into access errors, but it was some time ago. I removed the Sudos to check if they are still needed.